android/app/src/main/cpp/app/organicmaps/Framework.hpp

```diff
@@ -57,12 +57,10 @@ namespace android
     ::Framework m_work;
 
     math::LowPassVector<float, 3> m_sensors[2];
-    double m_lastCompass;
+    double m_lastCompass = 0;
```

Comment:
nit:
```suggestion
    double m_lastCompass{0};
```

Reply:
No, this is the stupidest syntax, imo.

Reply:
Preventing unexpected narrowing is important. It's a good habit.

> Use brace initialization to convert arithmetic types (e.g., int64_t{x}). This is the safest approach because code will not compile if conversion can result in information loss. The syntax is also concise. [source](https://google.github.io/styleguide/cppguide.html#Casting)

https://stackoverflow.com/a/18222927

https://www.modernescpp.com/index.php/initialization/